optimum/exporters/onnx/model_configs.py

```diff
@@ -728,6 +730,94 @@ def generate_dummy_inputs(self, framework: str = "pt"):
         return dummy_inputs
 
 
+class HubertOnnxConfig(AudioOnnxConfig):
+    NORMALIZED_CONFIG_CLASS = NormalizedConfig
+
+
+class Wav2Vec2OnnxConfig(HubertOnnxConfig):
+    pass
+
+
+class Wav2Vec2ConformerOnnxConfig(HubertOnnxConfig):
+    pass
+
+
+class SEWOnnxConfig(HubertOnnxConfig):
+    pass
+
+
+class SEWDOnnxConfig(HubertOnnxConfig):
+    pass
+
+
+class UniSpeechOnnxConfig(HubertOnnxConfig):
+    pass
+
+
+class UniSpeechSATOnnxConfig(HubertOnnxConfig):
+    pass
+
+
+class WavLMOnnxConfig(HubertOnnxConfig):
+    pass
+
+
+class ASTDummyAudioInputGenerator(DummyAudioInputGenerator):
```

Comment:
It seems that it is often required to overwrite the behavior of `DummyAudioInputGenerator` => Also needed [here](https://github.com/huggingface/optimum/pull/622/files#diff-cf7d52b6c0ee70bc45f40c2c40ac99b49dbf02f2fdb0a141e9483f764126e05eR809).

Maybe we can make it more generic?
WDTY @mht-sharma @fxmarty

Reply:
Possible did not look enough models when added for whisper.

AFAIK: There are 2 types of inputs in audio models

1. `input_features` (mel spectogram): Shapes (batch_size, feature_size, sequence_length) & (batch_size, sequence_length, feature_size).
2. `input_values` (raw speech): Shapes (batch_size, sequence_length)


In the current `AudioInputGenerator` we have `nb_max_frames` (terminology only used within feature extractor, docs refer to this as sequence length) for the sequence_length part of the `input_features`. Let's remove that and we can just use the `sequence_length` itself.

For getting the feature size, we can use something like: 
```python
NormalizedAudioConfig.with_args(
        feature_size="num_mel_bins", // or "input_features_per_channel"
    )
```

Handling `input_features` would be tricky as it has 2 shapes for same name. We can make one of them default and create new input generator for other shape when required?